android-app/app/src/main/res/values/themes.xml

```diff
@@ -49,15 +49,15 @@
         <item name="android:paddingBottom">@dimen/largeButtonVerticalPadding</item>
     </style>
 
-    <style name="Theme.FiBo.Button.DeleteUserButton" parent="Widget.Material3.Button">
+    <style name="Theme.FiBo.Button.SettingsButton" parent="Widget.Material3.Button">
         <item name="shapeAppearance">@style/ShapeAppearance.Material3.Corner.ExtraSmall</item>
         <item name="android:textSize">@dimen/standardTextSize</item>
         <item name="android:paddingTop">18dp</item>
         <item name="android:paddingBottom">18dp</item>
         <item name="cornerRadius">10dp</item>
         <!-- TODO: find right background color for delete user button -->
         <item name="backgroundTint">@color/top_bar_placeholder</item>
-        <item name="android:textColor">@color/delete_red</item>
+        <item name="android:textColor">@color/mainTextColor</item>
         <item name="android:textAlignment">textStart</item>
         <item name="android:layout_marginStart">5dp</item>
         <item name="android:layout_marginEnd">5dp</item>
```

Comment:
Personally I dont like this style, buuuut it is exactly like the mockup. If I remember correctly I'm not the only one who does not really like the way it looks so I'm not sure whether we should ignore it and tackle this issue later on or whether we should do it correctly already at the beginning. 

Nonetheless we got a ticket soon to apply M3 Material Design. Maybe we could include in that ticket to update these button accordingly to the Material Design. Ideally it should be enough to add a style attribute like style="@style/Widget.Material3.Button....." and maybe some shapeAppearance.

Because of that I think we should not tackle this issue in this PR 👍